posit_workbench_login_guidance.md

```diff
@@ -0,0 +1,39 @@
+# Guidance to log in and start a session on Posit Workbench
+
+Original Author: Ciara Gribben
+
+Last Updated: December 2022
+
+## Logging in to Posit Workbench
+1. Ensure you are connected to the VPN.
+
+2. Open a web browser (preferably Microsoft Edge).
```

Comment:
```suggestion
2. Open a web browser.
```
Edge might be IT's preference but I'd argue that we shouldn't put this. This might be me telling on myself but I don't think I've even tested using Edge.